### ISSUE TITLE
Deprecate /groups endpoint

### ISSUE THREAD

**Issue by znarf** [opencollective/opencollective#2037]
The `/groups/{slug}/backers|users` is deprecated and should not be used anymore. 

Unfortunately, it's still relied on by the following collectives (we see requests in the logs):

- openfl
- obsproject
- gulpjs 
- jest 
- webpack
- babel

We need to provide a compatible endpoint or ask these collectives to migrate to our GraphQL API v2.

**Comment by znarf** [comment]
Babel migrated in https://github.com/babel/website/pull/2031

**Comment by znarf** [comment]
@dodgepong OBS is the last one on my list. Can I help you get rid of this API call and replace it with a future proof version?

**Comment by Warchamp7** [comment]
@znarf I've migrated the OBS website to use the GraphQL API now

**Comment by znarf** [comment]
@Warchamp7 that's awesome to hear! is it live already?

The advantage of the REST endpoint approach compared to GraphQL is that it's cached on the CDN level and then faster. The data is formatted similarly.

**Comment by Warchamp7** [comment]
@znarf Yep, the change is already live. We shouldn't be hitting the REST endpoint anymore

**Comment by znarf** [comment]
@Warchamp7 I'm still seeing some requests to https://opencollective.com/api/groups/obsproject/backers in our logs. Do you know where that might be coming from?

**Comment by Warchamp7** [comment]
@znarf Does your logging say which page is making the request? Everything that was hitting it before should be pointed at GraphQL now

**Comment by znarf** [comment]
@Warchamp7 Sorry, I unfortunately don't have that information available.

**Comment by Warchamp7** [comment]
@znarf I had only commented out the fetch we had on some pages, which some bots may have still opted to go hit just cause it was in the page source. That URL no longer appears in any page now, commented or otherwise.

If anything is still hitting that endpoint, it is not us and we will not be affected by it disappearing.

**Comment by znarf** [comment]
Ok, maybe some cached versions somewhere, let's wait and see. Fyi, we definitely see less of these requests since yesterday, just not zero.

![Screenshot 2019-10-01 at 21 27 51](https://user-images.githubusercontent.com/806/65993917-7ae99f80-e492-11e9-923c-2d3e1f140956.png)
